docs/source/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+|
```

Comment:
is this line needed?

Reply:
That line just adds spacing underneath the header -- not necessarily needed. Do we want to remove?